src/utils/popper.ts

```diff
@@ -22,10 +23,96 @@ type VariationRtl =
   | "left-leading"
   | "left-trailing";
 
+export type PopperComputedPlacement = ComputedPlacement;
```

Comment:
Can you drop `Popper` from this interface to hide the implementation details? 

We may need to introduce another type to replace `PopperPlacement` too (this one is a separate item).

Reply:
Yeah we can rename all of these. Maybe when we move to floatingUI?